src/people.rs

```diff
@@ -997,6 +1130,49 @@ impl ContextPeopleExt for Context {
                 .insert(TypeId::of::<T>());
         }
     }
+
+    fn get_counts<T: Tabulator, F>(&self, tabulator: &T, print_fn: F)
+    where
+        F: Fn(&Context, &[String], usize),
+    {
+        let type_ids = tabulator.get_typelist();
+
+        // First, update indexes
```

Comment:
Since we have to do this here and in query_people, can we pull it out into a function?